src/app/icd/server/ICDConfigurationData.h

```diff
@@ -123,6 +125,8 @@ class ICDConfigurationData
 
     static constexpr System::Clock::Seconds32 kMaxIdleModeDuration = System::Clock::Seconds32(18 * kSecondsPerHour);
     static constexpr System::Clock::Seconds32 kMinIdleModeDuration = System::Clock::Seconds32(1);
+    // As defined in the spec, the maximum duration for the StayActiveDuration is 30s (spec 7.5.1)
+    static constexpr System::Clock::Milliseconds32 kMaxStayActiveDuration = System::Clock::Milliseconds32(30000);
```

Comment:
So what I see in "9.17.7.5.1. PromisedActiveDuration Field" is that 30s is a _floor_, not a ceiling?

That is, a server can always respond with a PromisedActiveDuration that's as large as it wants.  This thing needs better naming.  Maybe kMaxGuaranteedStayActiveDuration?